### PR TITLE
E2E test: seperate user session from local provider and meshery

### PR DESCRIPTION
**Pull request by jerensl**
**Notes for Reviewers**

Seperating the user sessions from meshery and local provider to prevent data race

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/playwright.config.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ module.exports = defineConfig({
         ...devices['Desktop Chrome'],
         provider: 'Meshery',
         // Use prepared auth state.
-        storageState: 'playwright/.auth/user.json',
+        storageState: 'playwright/.auth/user-local-provider.json',
```
Comment (yash37158) [inline ui/playwright.config.js:70]: @jerensl How about avoiding repeating the storage state path by defining it once and reusing it ?
Something as:

const storageStateMeshery = 'playwright/.auth/user-meshery-provider.json';
const storageStateNone = 'playwright/.auth/user-local-provider.json';

**Review by yash37158 — APPROVED** [review]
LGTM

**Review by hamza-mohd — DISMISSED** [review]
lgtm

**Comment by yash37158** [comment]
@jerensl  Could you please take a look at the ESLint check? It seems to be failing.

**Review by amitamrutiya — APPROVED** [review]

**Pull request merged**